### PR TITLE
#159268417 Distinguish between tools and technologies in README

### PR DESCRIPTION
**Pull request by olusoladavid**
#### What does this PR do?

Clarifies the distinction between technologies and tools 
#### Description of Task to be completed

- Understand the difference between a tool and a technology.
- Rewrite README file to reflect the newly acquired understanding

#### How should this be manually tested?

Review the changed files
#### What are the relevant pivotal tracker stories?

[#159268417](https://www.pivotaltracker.com/story/show/159268417)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,10 @@ MyDiary is an online journal where users can pen down their thoughts and feeling
 
 ## Technologies
 
-- Nodejs
-- Express
-- Mocha (with Chai as assertion library)
-- Babel (ES6+ to ES5 transpilation)
-- ESLint (with Airbnb style guide)
+- Nodejs (with ExpressJS)
```
Comment (andela-ookoro) [inline README.md]: NodeJS is not a technology, it is a runtime engine. TDD is a methodology, ES6 is a language specification and Linting is validating a code through a set of rules. 

The heading can be `Tools and Technologies`, then TDD, linting, and Es6 should be removed. Tools can be the database used, the library used (such as react), the engine used ( Nodejs, .net, Laravel), the language (PHP, Java, Js, Scala, Ruby), cloud services etc

Comment (andela-ookoro) [reply README.md]: These links would be helpful https://stackoverflow.com/questions/3057526/framework-vs-toolkit-vs-library and https://github.com/andela-ookoro/completenews

Comment (olusoladavid) [reply README.md]: Okay. That's revealing. I will make ammendments.

File: README.md [hunk 2]
```diff
@@ -18,13 +18,14 @@ MyDiary is an online journal where users can pen down their thoughts and feeling
...
-- Nodejs
-- Express
-- Mocha (with Chai as assertion library)
-- Babel (ES6+ to ES5 transpilation)
-- ESLint (with Airbnb style guide)
+## Tech Stack
+
+- [Node.js](https://nodejs.org/)
+- [Express.js](https://expressjs.com/)
+- [PostgreSQL](https://www.postgresql.org/)
```
Comment (andela-ookoro) [inline README.md]: Have you used PostgreSQL in this project, if no; remove it and add it when you have used it. 
The Readme file should be updated alongside the project.

**Review by andela-ookoro — CHANGES_REQUESTED** [review]
Review the comment on Tech Stack and share your thought

File: README.md (hunk 1 above)
Comment (olusoladavid) [reply README.md]: I understand better now. Thanks.

**Review by andela-ookoro — APPROVED** [review]
good job

**Pull request merged**